docroot/themes/custom/twentyeighteen/sass/components/403-404/403-404.twig

```diff
@@ -0,0 +1,9 @@
+<a class="call-to-action" href="#">
```

Comment:
@charlieartist I don't believe this file is necessary. Can we remove this? It doesn't look related to the work of this ticket. Please confirm if that's the case, and if not let me know why it's here.

Reply:
This was a placeholder file for adding a mockup of the 403 and 404 pages to the styleguide, which has been removed. SCSS file reference also updated on this.